projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -26,6 +26,15 @@ function wpcom_launchpad_should_use_wp_admin_link() {
  * @return Task[]
  */
 function wpcom_launchpad_get_task_definitions() {
+	$experiment_name  = 'calypso_signup_onboarding_goals_first_flow_holdout_v2_20250131';
+	$user             = wp_get_current_user();
+	$is_user_assigned = false;
+
+	if ( defined( 'IS_WPCOM' ) && IS_WPCOM && $user && function_exists( '\ExPlat\get_user_assignment' ) ) {
```

Comment:
We can also use `get_current_user_assignment`, and we'll also need to create a stub on wpcom (bin/teamcity-builds/jetpack-stubs/stub-defs.php) to let the phan issue go away.

Reply:
Oooh nvm I think it's better to check the `$user` value here.

Reply:
Stub added: 173369-ghe-Automattic/wpcom

Reply:
TIL